catroid/src/main/java/org/catrobat/catroid/io/BackwardCompatibleCatrobatLanguageXStream.java

```diff
@@ -620,6 +621,9 @@ private void initializeBrickInfoMap() {
 		brickInfo = new BrickInfo(HideTextBrick.class.getSimpleName());
 		brickInfoMap.put("hideTextBrick", brickInfo);
 
+		brickInfo = new BrickInfo(WriteVariableOnDeviceBrick.class.getSimpleName());
```

Comment:
this is unnecessary.
youre adding a new brick, theres no old version of it with a different name.

(see cameraBrick below, this is the *BackwrardCompatible"-bla class. in case you ever rename your brick after we had it released, you have to add it here to let Xstream know that the "videoBrick" in the xml is a "CameraBrick" in the codebase.)

Reply:
by adding it with the very same name, but with a small char at the beginning, although initially unnecessary, it would prevent us from ever again removing it from this list.

please do not add new bricks here.

Reply:
I will keep that in mind.